.github/workflows/build.yml

```diff
@@ -92,7 +92,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        variant: ["prod", "qa"]
+        variant: ["prod", "dev"]
```

Comment:
I tried to do this [here](https://github.com/bitwarden/android/pull/3616/files) but was told not too.

Has something changed?

Reply:
Should we update line 233 as well?

Reply:
Indeed